host/integration_test.go

```diff
@@ -3157,7 +3157,7 @@ func (s *integrationSuite) TestStickyTimeout_NonTransientDecision() {
 			// send a signal on third failure to be buffered, forcing a non-transient decision when buffer is flushed
 			/*if failureCount == 3 {
 				err := s.engine.SignalWorkflowExecution(createContext(), &types.SignalWorkflowExecutionRequest{
-					Domain:            common.StringPtr(s.domainName),
+					Domain:            s.domainName,
```

Comment:
commented out?

not relevant for this diff, but is this something we should restore?